# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gaim-otr/ui.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- gaim-otr-3.0.0/gtk-ui.c_orig	2005-11-07 03:05:52.000000000 +0100
+++ gaim-otr-3.0.0/gtk-ui.c	2005-11-07 03:08:01.000000000 +0100
@@ -634,8 +634,6 @@
     gtk_box_pack_start(GTK_BOX(vbox), ui_layout.scrollwin,
 	    TRUE, TRUE, 0);
 
-    otrg_gtk_ui_update_keylist();
-
     hbox = gtk_hbox_new(FALSE, 5);
     gtk_box_pack_start(GTK_BOX(vbox), hbox, FALSE, FALSE, 5);
 
@@ -682,6 +680,8 @@
     gtk_signal_connect(GTK_OBJECT(vbox), "destroy",
 	    GTK_SIGNAL_FUNC(ui_destroyed), NULL);
 
+    otrg_gtk_ui_update_keylist();
+
     /* Handle selections and deselections */
     gtk_signal_connect(GTK_OBJECT(ui_layout.keylist), "select_row",
 	    GTK_SIGNAL_FUNC(clist_selected), NULL);
